Docs changes suggestions after running onboarding tutorial. (#833)
MetaMask/metamask-docs: snaps/tutorials/gas-estimation.md

## Prerequisites

- [Snaps installed](../get-started/install-snaps.md)
- A text editor (for example, [VS Code](https://code.visualstudio.com/))
- [Yarn](https://yarnpkg.com/) version 3

## Steps

> ### 1. Set up the project

Reviewer: Can you make similar changes to step 1 of the transaction insights snap tutorial?

Author: I will take this up in a separate PR. You can approve this and I will merge it.

Author: This: https://github.com/MetaMask/metamask-docs/pull/835

Create a new snap project using the [`@metamask/create-snap` CLI](https://github.com/MetaMask/snaps/tree/main/packages/create-snap) by running:

```bash
yarn create @metamask/snap gas-estimation-snap
```

or

```bash
npm create @metamask/snap gas-estimation-snap
```

Next, `cd` into the `gas-estimation-map` project directory and run:

```bash
yarn install
```

> This will initialize your development environment with the required dependencies.

Reviewer: ```suggestion
This initializes your development environment with the required dependencies.
```

      "packageName": "snap",
      "registry": "https://registry.npmjs.org/"
   }
}
```

> Next, download [this free `gas.svg` icon file](https://raw.githubusercontent.com/Montoya/gas-fee-snap/main/packages/snap/images/gas.svg) into the `/packages/snap/images` folder.

Reviewer: ```suggestion
Next, download [this `gas.svg` icon file](https://raw.githubusercontent.com/Montoya/gas-fee-snap/main/packages/snap/images/gas.svg) into the `/packages/snap/images` folder.
```
Since free is mentioned in the next sentence.

To build and test your snap:

1. Open `package.json` in the root directory of the project, and increase the `"version"` (if the `"version"` is
    `0.1.0`, increase it to `0.2.0`).

2. From the command line, run `yarn start`.

>     On the Terminal, at the bottom of the message log, you will see the browser url address:

Reviewer: ```suggestion
    In the terminal, at the bottom of the message log, you see the browser URL:
```

    - **Access the internet.**
    - **Display dialog windows in MetaMask.**

5. Select **Approve & install**.

> 6. After install, the **Send message** button on the page will be enabled. Click on this **Send message** button. A dialog prompt displays with the response from the gas fee API:

Reviewer: ```suggestion
6. After installing, the **Send message** button on the page is enabled. Select this button. A dialog prompt displays with the response from the gas fee API:
```
